@xen-orchestra/backups/_backupWorker.mjs

```diff
@@ -178,8 +178,8 @@ process.on('message', async message => {
       const result = message.runWithLogs
         ? await Task.run(
             {
-              name: 'backup run',
-              onLog: data =>
+              properties: { name: 'backup run' },
+              onProgress: data =>
```

Comment:
AFAIK , onProgress can be emitted with a simple progress change. I think we should differenciate onProgress of type log and the other progresses